domains/mi-dominio.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "guardini-philama",
+    "email": "guardi2005@yahoo.fr"
+  },
+  "record": {
+    "CNAME": "https://portfoliovuejs.vercel.app/"
```

CNAME can't contain protocol or slashes
```suggestion
    "CNAME": "portfoliovuejs.vercel.app"
```